react/src/views/Box/BoxView.tsx

```diff
@@ -218,18 +219,18 @@ function BTBox() {
   const { isOpen: isPlusOpen, onOpen: onPlusOpen, onClose: onPlusClose } = useDisclosure();
   const { isOpen: isMinusOpen, onOpen: onMinusOpen, onClose: onMinusClose } = useDisclosure();
 
-  if (loading) {
-    return <APILoadingIndicator />;
-  }
   if (
+    loading ||
     updateBoxLocationMutationStatus.loading ||
     assignBoxToDistributionEventMutationStatus.loading ||
     unassignBoxFromDistributionEventMutationStatus.loading
   ) {
     return <APILoadingIndicator />;
   }
+
+  if (error) return <div />;
```

Comment:
Why not show an error toast here?

Reply:
Clarified: error handling in BoxView and BoxEdit should be refactored (in the style of BoxCreate)

Reply:
done in #542